docs/data/sdks/typescript-node/index.md

```diff
@@ -41,22 +41,8 @@ init(API_KEY, {
 ### Configuration
 
 ???config "Configuration Options"
-    | <div class="big-column">Name</div>  | Description | Default Value |
-    | --- | --- | --- |
-    |`apiKey`| Required. `string`. The apiKey of your project. | `null` |
-    |`flushIntervalMillis`| `number`. The amount of time waiting to upload the event to the server in millionseconds. | 10 seconds.|
-    |`flushQueueSize`| `number`. The maximum number of events that can be stored locally before forcing an upload.  | 200 events. |
-    |`flushMaxRetries`| `number`. The max retry limits. | 12 times.|
-    |`logLevel`| `LogLevel.None` or `LogLevel.Error` or `LogLevel.Warn` or `LogLevel.Verbose` or `LogLevel.Debug`. The log level. | `LogLevel.Warn` |
-    |`loggerProvider`| `Logger`. Implements a custom `loggerProvider` class from the Logger, and pass it in the configuration during the initialization to help with collecting any error messages from the SDK in a production environment.| `Amplitude Logger` |
-    |`minIdLength`| `number`. Overrides the minimum length of `user_id` & `device_id` fields. | `5` |
-    |`optOut`| `boolean`. If `optOut` is `true`, the event isn't sent to Amplitude's servers. | `false` |
-    |`serverUrl`| `string`. The server url events upload to. | `https://api2.amplitude.com/2/httpapi` |
-    |`serverZone`| `EU` or  `US`. Set Amplitude Server Zone, switch to zone related configuration. To send data to Amplitude's EU servers should configure to `EU` | `US` |
-    |`storageProvider`| `Storage<Event[]>`. Implements a custom `storageProvider` class from Storage. | `LocalStorage` |
-    |`useBatch`| `boolean`. When `true`, uses the Batch API instead of the HTTP V2 API.| `false` |
-    |`transportProvider`| `Transport`. Custom HTTP client. For example, sending requests to your proxy server with customized HTTP request headers.| `Transport` |
-    |`transport`| `TransportType.XHR` or `TransportType.SendBeacon` or `TransportType.Fetch`. Set the transport type. | `TransportType.Fetch` |
+--8<-- "includes/sdk-ts/shared-ts-configuration.md"
```

Comment:
Double configuration header
<img width="858" alt="Screen Shot 2023-06-09 at 12 21 59 PM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/452bb1bf-52a6-4797-b563-d5f26be49a48">

Reply:
Also, Node(server) SDK has different default values before.  We didn't change that since Node is still in 1.x, right?
https://www.docs.developers.amplitude.com/data/sdks/typescript-node/#configuration

Reply:
ahh didnt notice the default values. i can just revert this change